R/count_watershed_data.R

```diff
@@ -221,9 +225,46 @@ count_watershed_data <- function(data_dir,
 
         #-------------------------------------------------------
         # TELECONNECTION - WASTE WATER TREATMENT PLANTS
-        wasteflow_points[watersheds_select, ] -> wasteflow_select
+        wasteflow_points[watersheds_select, ] %>%
```

Comment:
This is no longer used, right? Best to remove redundancy in the code.

Reply:
Will do, I just wanted to make sure we liked the results of the new calculation before I got rid of it!